codalab/apps/teams/forms.py

```diff
@@ -1,14 +1,48 @@
 from django import forms
+from django.forms.widgets import ClearableFileInput, Input, CheckboxInput
+from django.utils.html import escape, conditional_escape
+from django.utils.encoding import force_unicode
 from .models import Team, TeamMembership
 from tinymce.widgets import TinyMCE
+from django.utils.html import format_html
+from django.utils.encoding import force_text
+from django.utils.safestring import mark_safe
 import os
 
+class CustomImageField(ClearableFileInput):
```

Comment:
This looks pretty awesome! Could someone submit a file named `<script>whatever()</script>` or something similar -- should we be worried about `mark_safe`? I don't think you can put the `/` in the filename so may not be a big deal..